config/common/character_interactions.cwt

```diff
@@ -18,16 +113,81 @@ character_interaction = {
     ## cardinality = 0..1
     can_be_picked = single_alias_right[trigger_clause]
     ## cardinality = 0..1
+    can_be_picked_title = single_alias_right[trigger_clause]
+    ## cardinality = 0..1
+    can_send_despite_rejection = bool
+    ## cardinality = 0..1
     is_highlighted = single_alias_right[trigger_clause]
-
+    ## cardinality = 0..1
+    highlighted_reason = localisation
+    
     ## cardinality = 0..1
     on_send = single_alias_right[effect_clause]
     ## cardinality = 0..1
     on_accept = single_alias_right[effect_clause]
     ## cardinality = 0..1
     on_decline = single_alias_right[effect_clause]
     ## cardinality = 0..1
+    auto_accept = bool
+    ## cardinality = 0..1
+    auto_accept = {
+        alias_name[trigger:custom_description] = alias_match_left[trigger:custom_description]
+    }
+    ## cardinality = 0..1
     on_auto_accept = single_alias_right[effect_clause]
+
+    ## cardinality = 0..1
+    popup_on_receive = bool
     ## cardinality = 0..1
-    ai_accept = single_alias_right[weight_block]
-}
\ No newline at end of file
+	pause_on_receive = bool
+    
+    ## cardinality = 0..1
+    populate_recipient_list = single_alias_right[effect_clause]
+    ## cardinality = 0..1
+    populate_actor_list = single_alias_right[effect_clause]
+    
+    ## cardinality = 0..1
+    ai_accept = single_alias_right[ai_chance_block]
+    ## cardinality = 0..1
+    ai_targets = {
+        ai_recipients = enum[ai_targets]
+        ## cardinality = 0..1
+        max = int_value_field
+    }
+    ## cardinality = 0..1
+    ai_frequency = int_value_field
```

Comment:
Can this actually be a variable?

Reply:
From what I can see, searching through CK3's code, it's always an integer.